servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/amqp/AmqpException.java

```diff
@@ -3,24 +3,22 @@
 
 package com.microsoft.azure.servicebus.amqp;
 
-import org.apache.qpid.proton.amqp.transport.*;
+
+import org.apache.qpid.proton.amqp.transport.ErrorCondition;
 
 /**
  * All AmqpExceptions - which EventHub client handles internally. 
  */
-public class AmqpException extends Exception
-{
+public class AmqpException extends Exception {
     private static final long serialVersionUID = -750417419234273714L;
-    private ErrorCondition errorCondition;
+    private transient ErrorCondition errorCondition;
```

Comment:
[SpotBugs-P3]
This Serializable class defines a non-primitive instance field which is neither transient, Serializable, or java.lang.Object, and does not appear to implement the Externalizable interface or the readObject() and writeObject() methods.  Objects of this class will not be deserialized correctly if a non-Serializable object is stored in this field.

Reply:
the ErrorCondition already not serializable,  Ignore it with using transient modifier to suppress the SpotBugs error